### ISSUE TITLE
Support signed distance on stl files via operator or expression

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#1295]
From Andy Cook and George Zagaris  -- we can use Quest (part of Axom) to do the signed distance. 

To summarize our conversation, it would be nice to have an option in VisIt to compute the signed distance function of an STL file. This would enable us to âshrink wrapâ multiple parts by:
Importing each .stl file
Translating/rotating each mesh in to the correct position
Computing the signed distance function of each mesh
Taking the union of the distance functions
Taking the zero isosurface
Outputing the isosurface âshrink wrapâ as a new stl file.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. As such, not all 
information was able to be captured in the transition. Below is
a complete record of the original redmine ticket.

Ticket number: 3019
Status: Pending
Project: VisIt
Tracker: Feature
Priority: High
Subject: Support signed distance on stl files via operator or expression
Assigned to: 
Category: 
Target version: 2.13.3
Author: Cyrus Harrison
Start: 02/22/2018
Due date: 
% Done: 0
Estimated time: 
Created: 02/22/2018 06:23 pm
Updated: 05/10/2018 08:26 pm
Likelihood: 
Severity: 
Found in version: 2.12.3
Impact: 3 - Medium
Expected Use: 3 - Occasional
OS: All
Support Group: Any
Description:
 From Andy Cook and George Zagaris  -- we can use Quest (part of Axom) to do the signed distance. 

To summarize our conversation, it would be nice to have an option in VisIt to compute the signed distance function of an STL file. This would enable us to âshrink wrapâ multiple parts by:
Importing each .stl file
Translating/rotating each mesh in to the correct position
Computing the signed distance function of each mesh
Taking the union of the distance functions
Taking the zero isosurface
Outputing the isosurface âshrink wrapâ as a new stl file.

Comments:
